labs/src/main/scala/org/scalalabs/basic/lab02/ListManipulationExercise01.scala

```diff
@@ -15,15 +15,14 @@ object ListManipulationExercise01 {
    * 
    */
   def firstElementInList[T](l: List[T]): T = {
-    //buildin
-    null.asInstanceOf[T]
+    l.head
```

Comment:
We _can_ omit the dot, but the style guide (see ["Suffix Notation"](http://docs.scala-lang.org/style/method-invocation.html)) actually prescribes against calling no argument functions this way because the compiler might accidentally treat `head` in this case as an infix operator in the case where there is more code below the line.